test/packages/good/manifest.yml

```diff
@@ -17,7 +17,8 @@ policy_templates:
         multi: false
         vars:
           - name: hosts
-            type: text
+            type: url
+            url_allowed_schemes: ['http', 'https']
```

Comment:
nit: sometimes we set values as "localhost:9876". Will such form be accepted/forbidden?

Reply:
Not by this config, but it could be accepted if configured as `['http', 'https', '']`. I have added a clarification about this in the spec.